Governance/appendix/glossary.md

```diff
@@ -0,0 +1,167 @@
+
+**CARE**
+  The CARE Principles for Indigenous Data Governance are people and purpose-oriented, reflecting the crucial role of data in advancing Indigenous innovation and self-determination. [See more](https://www.gida-global.org/care) 
+
+**CARE - Collective Benefit**
+  Data ecosystems shall be designed and function in ways that enable Indigenous Peoples to derive benefit from the data.
+
+**CARE - Authority to Control**
+  Indigenous Peoples’ rights and interests in Indigenous data must be recognised and their authority to control such data be empowered.
+
+**CARE - Responsibility**
+  Those working with Indigenous data have a responsibility to share how those data are used to support Indigenous Peoples’ self- determination and collective benefit.
+
+**CARE - Ethics**
+  Indigenous Peoples’ rights and wellbeing should be the primary concern at all stages of the data life cycle and across the data ecosystem.
+
+**CC**
+  [Creative Commons (CC)](https://creativecommons.org/) is a non-profit organisation that produces licenses to encourage sharing of knowledge, commonly used for data products. See also the [open access license](data-license) book page.
+
+**CDM**
+  Common Data Model (CDM) is an abstract data model for scientific datasets. This term can be used in different contexts. [Unidata’s Common Data Model](https://docs.unidata.ucar.edu/netcdf-java/5.3/userguide/common_data_model_overview.html) merges the netCDF, {term}`OPeNDAP`, and HDF5 data models to create a common API for many types of scientific data. The NetCDF Java library is an implementation of the CDM which can read many file formats besides netCDF. Another example of CDM is the one developed by [Copernicus](https://cds.climate.copernicus.eu/toolbox/doc/how-to/15_how_to_understand_the_common_data_model/15_how_to_understand_the_common_data_model.html) to provide a uniform description (conventions, structures, formats etc.) of all data and products in their Climate Data Store.
+
+**CDS**
+  [Copernicus Climate Data Store (CDS)](https://cds.climate.copernicus.eu/#!/home) provides access to a wide range of quality-assured climate datasets including observations, historical climate data records, estimates of Essential Climate Variables (ECVs) derived from Earth observations, global and regional climate reanalyses of past observations, seasonal forecasts and climate projections. It is an initiative of the European Climate Change Service.
+
+**CF conventions**
+  The CF Climate and Forecast conventions are used to set metadata attributes in netCDF files. [See more](../concepts/cf-conventions.md)
+
+**Citation**
+  Citation is the way you attribute a piece of work, it should contain all the information necessary to locate the original work.
+
+**CLEX**
+  The ARC Centre of Excellence for Climate Extremes
```

Comment:
Do you want to add a URL?

Reply:
I haven't quite finished all the acronyms etc, I would just merge for now, it was more about the format. I added some that popped into my mind at the time so I wouldn't forget them